packages/components/src/Banner/Banner.stories.tsx

```diff
@@ -25,10 +25,15 @@ type Args = React.ComponentProps<typeof Banner> & {
 const Template: Story<Args> = (args) => {
   const { content, heading, ...restArgs } = args;
   return (
-    <Banner {...restArgs}>
-      <Banner.Title as="h1">{heading}</Banner.Title>
-      <Banner.Body>{content}</Banner.Body>
-    </Banner>
+    <Banner
+      {...restArgs}
+      textContent={
+        <>
+          <Banner.Title as="h1">{heading}</Banner.Title>{" "}
+          <Banner.Body>{content}</Banner.Body>
```

Comment:
I wonder if it'd help also to rename this to `Banner.Message`? "Body" does seem to communicate there can be all sorts of things in there